claimedtasks.md

```diff
@@ -57,8 +57,42 @@ This is a **read-only** endpoint, the [POST to the claimed task](#post-method-si
 
 It returns the tasks claimed by the specified user
 
-## POST Method (collection level)
-The creation of claimed tasks is managed by the underline workflow system. No methods are exposed to manually trigger such creation to avoid workflow hjack and inconsistency.
+#### findAllByItem
+**/api/workflow/claimedtasks/search/findAllByItem?uuid=<:item-uuid>**
+It returns all the claimed tasks related to the specified item
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the item object
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only users with ADMIN right can use the endpoint
+
+#### findByItem
+**/api/workflow/claimedtasks/search/findByItem?uuid=<:item-uuid>**
+It returns, if any, the single task related to the specified item claimed by the current user
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the item object
+
+Return codes:
+* 200 OK - if the operation succeed
+* 204 No Content - if there is no claimed task for the specified item and the current user
+* 401 Unauthorized - if you are not authenticated
+
+## POST Method
+To create a claimedtask, i.e. to clam a poooltask. 
+The pooltask must be supplied as URI in the request body using the text/uri-list content-type
+If succeed a 201 code will be returned and the new claimedtask and the pooltask will removed.
```

Comment:
This sentence seems incomplete. I think it should say:

`If successful a 201 code will be returned along with the new claimedtask. The pooltask will also be removed.`

(Currently it seems to say the claimedtask will be removed, which isn't correct)